OpenRA.Game/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -224,6 +256,14 @@ public override bool HandleKeyPress(KeyInput e)
 			return false;
 		}
 
+		static IEnumerable<Actor> SelectActorsInBoxWithDeadzone(World world, int2 a, int2 b, Func<Actor, bool> cond)
+		{
+			if (a == b || (a - b).Length > 50)
```

Comment:
I'll create a constant for it. In the future I'd like to have a slider for it since that seems like a reasonable thing to do.